olp-cpp-sdk-core/tests/cache/DefaultCacheTest.cpp

```diff
@@ -369,4 +369,266 @@ TEST(DefaultCacheTest, ValueGreaterThanMemCacheLimit) {
                          content.begin()));
 
   cache.Close();
+}
+
+TEST(DefaultCacheTest, EvictionPolicykNone) {
+  olp::cache::CacheSettings settings;
+  settings.disk_path_mutable = olp::utils::Dir::TempDirectory() + "/unittest";
+  settings.max_memory_cache_size = 0;
+  settings.eviction_policy = olp::cache::EvictionPolicy::kNone;
+  {
+    SCOPED_TRACE("Put/Get decode.");
+    olp::cache::DefaultCache cache(settings);
+    ASSERT_EQ(olp::cache::DefaultCache::Success, cache.Open());
+    ASSERT_TRUE(cache.Clear());
+    std::string key1DataString{"this is key1's data"};
```

Comment:
Here camel-case, below snake-case. Please follow the coding style!
Additionally, I don't see any key2 and key3 and so on. Why do you need to call the var key1?

Reply:
Oops, my bad.